Makefile

```diff
@@ -89,7 +89,7 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: gowork controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
```

Comment:
Just curious: why has `gowork` been added here?

Reply:
I added it while i was experimenting with `https://github.com//dnephin/pre-commit-golang`. Thanks for pointing out. I'll remove it. Surely it is not needed there.